docs/user_docs/getting-started/usecases/deliver-hybrid-resources-project.md

```diff
@@ -2,18 +2,16 @@
 sidebar_position: 2
 ---
 
-# Deliver Project with Hybrid Resources
+# Deliver Wordpress Application on Kubernetes and Clouds
```

Comment:
change this file name with the new title

Reply:
a little grammar tip: Deliver the WordPress application on Kubernetes and clouds